spec.html

```diff
@@ -42836,6 +43231,32 @@ <h1>
           1. Return *true*.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-hostgrowsharedarraybuffer" type="host-defined abstract operation">
+        <h1>
+          HostGrowSharedArrayBuffer (
+            _buffer_: a SharedArrayBuffer,
+            _newByteLength_: a non-negative integer,
+          ): either a normal completion containing either ~handled~ or ~unhandled~, or a throw completion
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It gives the host an opportunity to perform implementation-defined growing of _buffer_. If the host chooses not to handle resizing of _buffer_, it may return ~unhandled~ for the default behaviour.</dd>
```

Comment:
```suggestion
          <dd>It gives the host an opportunity to perform implementation-defined growing of _buffer_. If the host chooses not to handle growing of _buffer_, it may return ~unhandled~ for the default behaviour.</dd>
```

Either works, but it's probably better to use the same verb here.

Reply:
Done.